src/main/scala/ie/zalando/fabric/gateway/web/marshalling/JsonModels.scala

```diff
@@ -240,7 +269,7 @@ trait JsonModels {
     Encoder.forProduct2("status", "children")(resp => (resp.status, resp.desiredIngressDefinitions))
 
   implicit val decodeResourceDetails: Decoder[ResourceDetails] = (c: HCursor) =>
-    Right(ResourceDetails(c.keys.map(_.toList).getOrElse(Nil)))
+    c.as[Map[String, Map[HttpVerb, Json]]].map(paths => ResourceDetails(paths))
```

Comment:
I don't really know why the ResourceDetails model was changed. What was the reason?

Reply:
I needed access to the http verbs